tfhe/src/high_level_api/keys/inner.rs

```diff
@@ -179,21 +231,57 @@ impl IntegerCompressedServerKey {
                    "
         );
         let key = crate::integer::CompressedServerKey::new_radix_compressed_server_key(integer_key);
-        Self { key }
+
+        let compression_keys = client_key.compression_key.as_ref().map_or_else(
+            || None,
+            |a| {
+                Some(
+                    client_key
+                        .key
+                        .key
+                        .new_seeded_compression_decompression_keys(a),
```

Comment:
naming of the function here as well for seeded -> compressed

Reply:
Done